locator/tablets.hh

```diff
@@ -228,6 +228,27 @@ struct resize_decision {
     sstring name() const;
 };
 
+struct table_load_stats {
+    uint64_t size_in_bytes = 0;
+    // Stores the minimum timestamp among all replicas, as coordinator wants to know if
+    // all replicas have completed splitting, which happens when they all store the
+    // timestamp of the current split decision.
+    api::timestamp_type split_ready_phase = api::max_timestamp;
```

Comment:
btw. it could be a sequence number, since it's only incremented by the topology coordinator.

Reply:
One of the nice things of having timestamp type is that we can use system.tablets output to know when resize started. What about we rename it to resize_generation (which uses timestamp as its type)?

/cc @tgrabiec